profiles/arpanaditya.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/75975428/149463291-061db5e2-5228-418b-b70f-21ae7f0b75ad.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/149539028-f6ece6db-370b-4b30-adcf-322668957435.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/149539028-f6ece6db-370b-4b30-adcf-322668957435.png",
```

Reply:
Yes, Actually I was thinking about this. But thanks for doing this. :)